feat(content): Korath Expansion Stage II - World-Ships (#7088)
endless-sky/endless-sky: data/korath/korath weapons.txt

		"missile strength" 60
		submunition "Firelight Turning"
		"stream"

> 	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels."

Author: More or less

> 	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels." 

Reviewer: ```suggestion
	description "As the Firestorm torpedo does not require full reactor shielding or safety interlocks, Korath engineers were able to further scale down the reactor element to fit in a missile casing for use on and against smaller vessels."
```

	"random angle" 360
	"random velocity" 2
	"velocity scale" 0.1

outfit "Firestorm Torpedo"
	plural "Firestorm Torpedoes"
	category "Ammunition"
	cost 19800
	thumbnail "outfit/firestorm torpedo"
	"mass" 0.7
	"firestorm torpedo capacity" -1
	"energy generation" 0.06
	"heat generation" 0.3

> 	description `Ammo for the Firestorm Turret.`

Reviewer 2: Should this description be beefed up?

Reviewer 2: ```suggestion
	description `More massive in both size and destructive capability, the Firestorm Torpedo dwarfs comparable weaponry. Its massive blast radius makes up for its lack of more recent Korath innovations. The Firestorm Torpedo is used as ammo for the Firestorm Turret.`
```

> 	description `More massive in both size and destructive capability, the Firestorm Torpedo dwarfs comparable weaponry. Its massive blast radius makes up for its lack of more recent Korath innovations. The Firestorm Torpedo is used as ammo for the Firestorm Turret.`

Reviewer 3: ```suggestion
	description `More massive in both size and destructive capability, the Firestorm Torpedo dwarfs comparable weaponry. Its massive blast radius makes it a dangerous part of larger ship arsenals, despite the lack of more recent Korath innovations. The Firestorm Torpedo is used as ammo for the Firestorm Turret.`
```

> 	description `More massive in both size and destructive capability, the Firestorm Torpedo dwarfs comparable weaponry. Its massive blast radius makes it a dangerous part of larger ship arsenals, despite the lack of more recent Korath innovations. The Firestorm Torpedo is used as ammo for the Firestorm Turret.`

Reviewer 2: ```suggestion
	description `More massive in both size and destructive capability, the Firestorm Torpedo dwarfs comparable weaponry. Its enormous blast radius makes it a dangerous part of larger ship arsenals, despite the lack of more recent Korath innovations. The Firestorm Torpedo is used as ammo for the Firestorm Turret.`
```

	"mass" 6
	"outfit space" -20
	"firestorm torpedo capacity" 13
	ammo "Firestorm Torpedo"
	description "This appears to be a modified Korath Mine Rack, designed to carry Firestorm Torpedoes."

outfit "Firestorm Turret"
	category "Secondary Weapons"
	cost 2383000
	thumbnail "outfit/firestorm turret"
	"mass" 20
	"outfit space" -48
	"weapon capacity" -48
	"turret mounts" -1
	"required crew" 1
	"firestorm torpedo capacity" 13
	"energy capacity" 1300
	weapon
		sprite "projectile/firestorm torpedo"
			"frame rate" 2
			"no repeat"
		"hardpoint sprite" "hardpoint/firestorm turret"
		sound "firestorm"
		ammo "Firestorm Torpedo"
		"turret turn" 2.7
		icon "icon/firestorm torpedo"
		"hit effect" "firestorm ring" 40
		"die effect" "small explosion"
		stream
		"reload" 210
		"firing energy" 635
		"firing heat" 451
		"inaccuracy" 3
		"lifetime" 1900
		"velocity" 6.6
		"acceleration" .6
		"drag" .12
		"turn" 1.6
		"homing" 4
		"infrared tracking" .7
		"optical tracking" .9
		"trigger radius" 40
		"blast radius" 95
		"shield damage" 2760
		"hull damage" 2760
		"heat damage" 5320
		"burn damage" 40
		"corrosion damage" 5
		"hit force" 4000
		"missile strength" 115

> 	description "This torpedo contains an augmented Korath Plasma Core reactor element, with no protective casing. When it activates, there is a tremendous explosion of heat and force, pushing ships away and even melting hulls."

Reviewer 3: ```suggestion
	description "Though not as refined as the newer Firelight version, this torpedo contains an augmented Korath Plasma Core reactor element completely stripped of its protective casing and safety interlocks. When activated, it generates a tremendous explosion of heat and force that pushes ships away and will even melt their hulls."
```

	"random angle" 360
	"random velocity" 2
	"velocity scale" 0.1

outfit "Firestorm Torpedo"
	plural "Firestorm Torpedoes"
	category "Ammunition"
	cost 19800
	thumbnail "outfit/firestorm torpedo"

> 	"mass" 0.7

Author: ```suggestion
	"mass" 1
```

	"firestorm torpedo capacity" -1
	"energy generation" 0.06
	"heat generation" 0.3

> 	description `More massive in both size and destructive capability, the Firestorm Torpedo dwarfs comparable weaponry. Its enormous blast radius makes it a dangerous part of larger ship arsenals, despite the lack of more recent Korath innovations. The Firestorm Torpedo is used as ammo for the Firestorm Turret.`

Author: ```suggestion
	description `Massive in both size and destructive capability, the Firestorm Torpedo's enormous blast radius makes it a dangerous part of larger ship arsenals, despite the lack of more recent Korath innovations. The Firestorm Torpedo is used as ammo for the Firestorm Turret.`
```

outfit "Firestorm Torpedo Rack"
	plural "Firestorm Torpedo Racks"
	category "Ammunition"
	cost "135000"
	thumbnail "outfit/firestorm torpedo rack"

> 	"mass" 6
> 	"outfit space" -20
> 	"firestorm torpedo capacity" 13

Author: ```suggestion
	"mass" 7
	"outfit space" -20
	"firestorm torpedo capacity" 13
```

	ammo "Firestorm Torpedo"
	description "This appears to be a modified Korath Mine Rack, designed to carry Firestorm Torpedoes."

outfit "Firestorm Turret"
	category "Secondary Weapons"
	cost 2383000
	thumbnail "outfit/firestorm turret"

> 	"mass" 20
> 	"outfit space" -48
> 	"weapon capacity" -48
> 	"turret mounts" -1
> 	"required crew" 1
> 	"firestorm torpedo capacity" 13
> 	"energy capacity" 1300

Author: ```suggestion
	"mass" 35
	"outfit space" -48
	"weapon capacity" -48
	"energy capacity" 1300
	"turret mounts" -1
	"required crew" 1
	"firestorm torpedo capacity" 13
```

	weapon
		sprite "projectile/firestorm torpedo"
			"frame rate" 2
			"no repeat"
		"hardpoint sprite" "hardpoint/firestorm turret"
		sound "firestorm"
		ammo "Firestorm Torpedo"
		"turret turn" 2.7
		icon "icon/firestorm torpedo"
		"hit effect" "firestorm ring" 40
		"die effect" "small explosion"
		stream
		"reload" 210
		"firing energy" 635
		"firing heat" 451
		"inaccuracy" 3
		"lifetime" 1900
		"velocity" 6.6
		"acceleration" .6
		"drag" .12
		"turn" 1.6
		"homing" 4
		"infrared tracking" .7
		"optical tracking" .9
		"trigger radius" 40
		"blast radius" 95
		"shield damage" 2760
		"hull damage" 2760
		"heat damage" 5320
		"burn damage" 40
		"corrosion damage" 5
		"hit force" 4000
		"missile strength" 115

> 	description "Though not as refined as the newer Firelight version, this torpedo contains an augmented Korath Plasma Core reactor element completely stripped of its protective casing and safety interlocks. When activated, it generates a tremendous explosion of heat and force that pushes ships away and will even melt their hulls."

Reviewer 4: Aren't those plasma cores fusion reactors? Do those have 'elements' that might be within even an order of magnitude of these missiles?

Author: Why not?  I'm not going to pretend to understand how a future alien sci-fi fusion reactor works, but I count 36 fins in Gefu's sprite.   Maybe each of those fins is pulling heat from one or two "elements" in the ring?
